docs/software/switch-config.md

```diff
@@ -1,114 +1,162 @@
-
-You can still use a 3-position switch to arm! AUX1 just needs to be 2-position, not the physical switch. Simply adjust the mixer in your transmitter so AUX1 works like an on-off switch. If your 3-position arm switch had a second function as well, such as enabling Blackbox, just use one of the other 7 AUX channels to send the switch on a second channel.
-
-### Why can't I have switches / potentiometers with full 10-bit resolution?
-
-A lot of the magic of ExpressLRS comes from its small packet sizes, there simply is not enough space to put all that data. There are tricks that can be done with interleaving sticks and switches packets but we believe sticks should be in every packet for the lowest control latency. 
-
-With the new [Full Resolution Switch Modes](#full-resolution-switch-modes) this is now possible.
-
-### What about "Normal" one bit switch mode?
-
-In 1.0, there was also a switch mode where there were 8x 1-position switches sent in every packet. This mode was removed in 2.0 due to its unpopularity and the flash space was used for other features.
-
-### Every time I change switch mode in Lua, it changes back!
-
-The switch mode can only be changed when a receiver is not connnected, to ensure a consistency between the RX and TX's interpretetation of the switch data. Power down your receiver, wait for the "Telemetry Lost" callout, and the switch mode change will stick.
+---
+template: main.html
+description: To optimize the performance of ExpressLRS, it has different Switch Modes explained in detail here.
+---
+
+<img src="https://raw.githubusercontent.com/ExpressLRS/ExpressLRS-Hardware/master/img/software.png">
+
+# Switch Configs
+
+!!! warning "WARNING" 
+    **Put your arm switch on AUX1**, and set it as **~1000 is disarmed, ~2000 is armed**.
+
+## Summary of Switch Configuration Modes
+
+This table summarizes the switch configuration modes, available channel switch positions and resolutions, channel update rate, and packet rate as it applies to each channel or flight controller auxiliary channel (Aux). Below the table are descriptions for each option.
+
+| Channel | Flight <br>Controller | Hybrid | Wide | Full Res <br>8ch | Full Res <br>16ch Rate/2 | Full Res <br>12ch Mixed |
+|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+| 1   | Sticks | **Normal<br>Range** | **Normal<br>Range** | **CRSF Ext<br>Limits** | CRSF Ext<br>Limits<sup>H</sup> | **CRSF Ext<br>Limits** |
+| 2   | Sticks | **Normal<br>Range** | **Normal<br>Range** | **CRSF Ext<br>Limits** | CRSF Ext<br>Limits<sup>H</sup> | **CRSF Ext<br>Limits** |
+| 3   | Sticks | **Normal<br>Range** | **Normal<br>Range** | **CRSF Ext<br>Limits** | CRSF Ext<br>Limits<sup>H</sup> | **CRSF Ext<br>Limits** |
+| 4   | Sticks | **Normal<br>Range** | **Normal<br>Range** | **CRSF Ext<br>Limits** | CRSF Ext<br>Limits<sup>H</sup> | **CRSF Ext<br>Limits** |
+| **5** | **Aux 1** | **2-pos<br>Arm** | **2-pos<br>Arm** | **2-pos<br>Arm** | **2-pos<br>Arm** | **2-pos<br>Arm** |
```

Comment:
16ch fullres CH5 is CRSF Ext Limits but also Arm still (and not sent in every packet)
`CRSF Ext<br>Limits<sup>H</sup><br/>Arm`?